src/ceph-volume/ceph_volume/api/lvm.py

```diff
@@ -376,8 +376,12 @@ def set_tags(self, tags):
             self.set_tag(k, v)
         # after setting all the tags, refresh them for the current object, use the
         # pv_* identifiers to filter because those shouldn't change
-        pv_object = self.get_first_pv(filter={'pv_name': self.pv_name,
-                                              'pv_uuid': self.pv_uuid})
+        pv_object = self.get_single_pv(filter={'pv_name': self.pv_name,
+                                               'pv_uuid': self.pv_uuid})
+
+        if not pv_object:
+            raise(RuntimeError('No PV was found.'))
```

Comment:
nit: no need for parentheses -
```suggestion
            raise RuntimeError('No PV was found.')
```